ui.frontend/src/view/FormPanel.js

```diff
@@ -43,6 +43,11 @@ export default class FormPanel extends FormFieldBase {
             instanceManager.addChild(this);
             this.setInstanceManager(instanceManager);
         }
+        this.syncLayoutHtmlWithModel();
```

Comment:
we are misusing setModel API. This API should only be used to set the model in the view. Any other thing being done will be violating this principle.

Reply:
Hi @nit23uec , In FormField, we are using the setModel to call syncHtmlWithModel for all the components. The setters can be used theoretically for purposes of pre setting up of value and post setting the values as well.
Like scenarios where we validate something before setting or convert something to a different format before setting.
We can discuss if this still needs to be removed .